src/VirtualClient/VirtualClient.Monitors/ExecuteCommandMonitor.cs

```diff
@@ -118,6 +118,17 @@ public string MonitorEventType
             }
         }
 
+        /// <summary>
+        /// Parameter defines Telemetry Splitting (true/false). Default = false.
+        /// </summary>
+        public bool TelemetrySplit
```

Comment:
Per earlier comments. Not necessary. Just perform the split if the size of the content exceeds the max chars.